classes/ActionScheduler_NonexecutableAction.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Class ActionScheduler_NonexecutableAction
+ *
+ * Stored action which should not be executed because it has already been executed, and has the
+ * failed or completed status, or becuase it has been cancelled, and has the cancelled status.
+ */
+class ActionScheduler_NonexecutableAction extends ActionScheduler_StoredAction {
```

Comment:
This should be `ActionScheduler_Nonexecutable_Action` for consistency.

Reply:
@JPry see other responses RE: consistency.